scanners/nikto/scanner/Dockerfile

```diff
@@ -2,6 +2,19 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM localhost:5000/sullo/nikto
+FROM alpine:3.14 as build
+ARG scannerVersion
+RUN apk add git
+RUN git clone --depth 1 --branch "nikto-$scannerVersion" https://github.com/sullo/nikto.git /nikto
+
+FROM alpine:3.14
+ENV  PATH=${PATH}:/nikto
+COPY --from=build /nikto/program /nikto
 COPY wrapper.sh /wrapper.sh
+RUN apk add --update --no-cache --virtual .build-deps \
+     perl \
+     perl-net-ssleay \
+  && addgroup nikto \
+  && adduser -G nikto -s /bin/sh -D nikto \
+  && chown -R nikto.nikto /nikto
```

Comment:
Is the user not actually used in the image?
Also should the user have a numeric user id?

Is it possible to avoid the chown here by running it during the copy?: https://docs.docker.com/engine/reference/builder/#copy
Otherwise the chowned contents are duplicated in two layers.